trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './base_decorator'
+
+class Trimmer < Decorator
+  def correct_name
+    super()
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[1...10] : @nameable.correct_name
```

Comment:
- [ ] Your trimmer method starts trimming from the second letter, instead the first. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/81468675/232842172-09fb6d03-6aea-4dac-98ff-1e938d43d035.png)

  You can use this

  ```
  [0...10]
  ```